pkg/storage/storage.go

```diff
@@ -668,3 +679,70 @@ func (s *storageModule) Monitor(ctx context.Context) <-chan pkg.PoolsStats {
 
 	return ch
 }
+
+func (s *storageModule) periodicallyCheckDiskShutdown() {
+	ticker := time.NewTicker(5 * time.Minute)
+	quit := make(chan struct{})
+	go func() {
+		for {
+			select {
+			case <-ticker.C:
+				log.Info().Msg("Checking pools for disks that should be shutdown...")
+				s.shutdownDisks()
+			case <-quit:
+				ticker.Stop()
+				return
+			}
+		}
+	}()
+}
+
+// shutdownDisks will check the disks power status.
+// If a disk is on and it is not mounted then it is not supposed to be on, turn it off
+func (s *storageModule) shutdownDisks() {
+	for _, pool := range s.pools {
+		for _, device := range pool.Devices() {
+			log.Debug().Msgf("checking device: %s", device.Path)
+			on, err := checkDiskPowerStatus(device.Path)
+			if err != nil {
+				log.Err(err).Msgf("error occurred while checking disk power status")
+				return
+			}
+
+			_, mounted := pool.Mounted()
+			if mounted || !on {
+				continue
+			}
+
+			log.Debug().Msgf("shutting down device %s because it is not mounted and the device is on", device.Path)
+			err = pool.Shutdown()
+			if err != nil {
+				log.Err(err).Msgf("failed to shutdown device %s", device.Path)
+				return
```

Comment:
Same comment, if i can't handle one disk it doesn't mean i never try again (and what about the other ones i can shutdown?)